pkg/controller/main-controller.go

```diff
@@ -1348,7 +1348,7 @@ func (c *Controller) syncHandler(key string) (Result, error) {
 	// Stay in this state until minio is ready
 	if tenant.Status.HealthStatus != miniov2.HealthStatusGreen {
 		c.updateTenantStatus(ctx, tenant, StatusWaitingMinIOIsHealthy, 0)
-		c.recorder.Event(tenant, corev1.EventTypeWarning, WaitingMinIOIsHealthyReason, fmt.Sprintf("Waiting for MinIO to be ready: %s", err))
+		c.recorder.Event(tenant, corev1.EventTypeWarning, WaitingMinIOIsHealthyReason, fmt.Sprintf("Waiting for MinIO(%s/%s) to be ready", tenant.Namespace, tenant.Name))
```

Comment:
No need to log the namespace and name. It's logged in the tenant itself, so the namespace/name is already known.
```suggestion
		c.recorder.Event(tenant, corev1.EventTypeWarning, WaitingMinIOIsHealthyReason, "Waiting for MinIO to be ready")
```